msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -681,8 +650,11 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 		/>
 	</Target>
 
-	<Import Project="$(MSBuildThisFileDirectory)Xamarin.Mac.XM45.targets"
-		Condition="'$(UseXamMacFullFramework)' == 'true'"/>
+	<Import Project="$(MSBuildThisFileDirectory)Xamarin.Mac.XM45.xbuild.targets"
+		Condition="'$(UseXamMacFullFramework)' == 'true' And '$(IsXBuild)' == 'true' "/>
+
+	<Import Project="$(MSBuildThisFileDirectory)Xamarin.Mac.msbuild.targets"
+		Condition="'$(IsXBuild)' == 'false' "/>
```

Comment:
The usual pattern is - `'$(IsXBuild)' != 'true'`, that way the default/empty value "means" `false`. Just a nitpick.